BUG: Fix potential segfault after pd.Categorical(pd.Series(...), categories=...) (#25368)
pandas-dev/pandas: doc/source/whatsnew/v0.24.2.rst

- Fixed regression in :meth:`DataFrame.duplicated()`, where empty dataframe was not returning a boolean dtyped Series. (:issue:`25184`)
- Fixed regression in :meth:`Series.min` and :meth:`Series.max` where ``numeric_only=True`` was ignored when the ``Series`` contained ```Categorical`` data (:issue:`25299`)

> - Fixed regression in ``Categorical``, where constructing it from a categorical ``Series`` and an explicit ``categories=`` that differed from that in the ``Series`` could create a object with nonsense data. In particular, the ``Categorical`` could have out-of-bounds ``codes`` which would trigger segfaults elsewhere in Pandas. (:issue:`25318`)

Reviewer: ```suggestion
- Fixed regression in ``Categorical``, where constructing it from a categorical ``Series`` and an explicit ``categories=`` that differed from that in the ``Series`` created an invalid object which could trigger segfaults. (:issue:`25318`)
```

Thanks for the update! That's clearer, but also quite long, so a suggestion to make it a bit shorter

- Fixed regression in :class:`TimedeltaIndex` where `np.sum(index)` incorrectly returned a zero-dimensional object instead of a scalar (:issue:`25282`)
- Fixed regression in ``IntervalDtype`` construction where passing an incorrect string with 'Interval' as a prefix could result in a ``RecursionError``. (:issue:`25338`)

> - Fixed regression in ``Categorical``, where constructing it from a categorical ``Series`` and an explicit ``categories=`` that differed from that in the ``Series`` created an invalid object which could trigger segfaults. (:issue:`25318`)

Reviewer 2: use ``:class:`Categorical` ``

**Categorical**

> - Constructing a `Categorical` from a `Series` could cause segfaults (:issue:`25318`)

Reviewer: Can you move this to the regressions section above?

Reviewer 2: be more specific about this it is too general a statement
